.github/workflows/crowdx-tests.yml

```diff
@@ -0,0 +1,36 @@
+name: CrowdX module tests
+
+on:
+  push:
+    branches:
+      - 'main'
+      - '*github-action*'
+
+  pull_request:
+    branches:
+      - '*'
```

Comment:
```suggestion
  push:
    branches:
      - 'main'
      - '*github-action*'
    paths:
      - 'modules/crowdx'

  pull_request:
    branches:
      - '*'
    paths:
      - 'modules/crowdx'
```

Reply:
Considering this will be a monorepo, you could make use of a paths filter so the library isn't tested when it shouldn't

Reply:
You are completely right, I commited your changes.